package.json

```diff
@@ -84,7 +84,7 @@
     "ember-sinon-qunit": "3.2.0",
     "ember-source": "2.18.2",
     "ember-truth-helpers": "2.0.0",
-    "ember-validations": "HospitalRun/ember-validations#2.0.0",
+    "ember-validations": "2.0.0-alpha.5",
```

Comment:
That's a workaround. Shouldn't be there

Reply:
I forgot I put that in there. I'll remove it.

Reply:
Done.